site/app/controllers/admin/SqlToolboxController.php

```diff
@@ -58,4 +58,70 @@ public function runQuery(): JsonResponse {
             $this->core->getCourseDB()->rollback();
         }
     }
+
+    #[Route("/courses/{_semester}/{_course}/saveQuery", methods: ["POST"])]
+    public function saveQuery(): string {
```

Comment:
These should return `JsonResponse` and return a success or failure rather than a string